amp/TEMPLATE/ampTemplate/dashboard/dev/app/js/app/views/chart-aid-predictability.js

```diff
@@ -24,11 +24,23 @@ module.exports = ChartViewBase.extend({
       var key = $(e.currentTarget).find('.nv-legend-text').text();
 	  var plannedDisbursementTrn = app.translator.translateSync("amp.dashboard:aid-predictability-planned-disbursements","Planned Disbursements");
 	  var actualDisbursementTrn = app.translator.translateSync("amp.dashboard:aid-predictability-actual-disbursements","Actual Disbursements");
-	  if(key == plannedDisbursementTrn){
-		  this.model.set('showPlannedDisbursements', !this.model.get('showPlannedDisbursements'));	
-	  }else if(key == actualDisbursementTrn){
-		  this.model.set('showActualDisbursements', !this.model.get('showActualDisbursements'));	
+	  var planned = this.model.get('showPlannedDisbursements');
```

Comment:
In order to start improving some old dashboard code, could you replace the == with ===?